project.clj

```diff
@@ -1,4 +1,4 @@
-(def tk-version "1.3.0")
+(def tk-version "1.3.1-SNAPSHOT")
```

Comment:
This needs to be fixed; I added a bootstrap testutil to TK that I used in the tests in this PR.